integrations/sklearn/Neptune-Scikit-learn.ipynb

```diff
@@ -0,0 +1,2048 @@
+    "* [logged test predictions](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/artifacts?path=csv%2F&file=test_predictions.csv),\n",
+    "* [logged test scores](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/charts),\n",
+    "* [logged regressor visualizations](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/logs) - look for \"charts_sklearn\",\n",
+    "* [logged metadata](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/details) including git summary info."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Scikit-learn classification"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 1: Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import load_digits\n",
+    "from sklearn.ensemble import GradientBoostingClassifier\n",
+    "from sklearn.model_selection import train_test_split"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 2: Create and fit gradient boosting classifier"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "gbc = GradientBoostingClassifier(n_estimators=120, learning_rate=0.12, min_samples_split=3, min_samples_leaf=2)\n",
+    "\n",
+    "X, y = load_digits(return_X_y=True)\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20, random_state=28743)\n",
+    "\n",
+    "gbc.fit(X_train, y_train)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AW4MM4qj5ErX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 3: Create Neptune experiment and log regressor summary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 71
+    },
+    "colab_type": "code",
+    "id": "GvDSBSrOx-R4",
+    "outputId": "667d1b2f-cdef-44f0-f505-832205dea223",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import neptune\n",
+    "from neptunecontrib.monitoring.sklearn import log_classifier_summary\n",
+    "\n",
+    "neptune.init('shared/sklearn-integration',\n",
+    "             api_token='ANONYMOUS')\n",
+    "exp = neptune.create_experiment(name='classification-example',\n",
+    "                                tags=['GradientBoostingClassifier', 'classification'])\n",
+    "\n",
+    "log_classifier_summary(gbc, X_train, X_test, y_train, y_test, experiment=exp)"
```

Comment:
I think we should add a comment with what was logged and that the test prediction .csv can be huge

Reply:
It will be 1k lines. Added comment what is logged